src/components/media-loader.js

```diff
@@ -78,9 +74,7 @@ AFRAME.registerComponent("media-loader", {
         );
         this.el.addEventListener("model-error", this.onError, { once: true });
         this.el.setAttribute("gltf-model-plus", {
-          src: raw,
-          contentType,
-          basePath: THREE.LoaderUtils.extractUrlBase(origin),
+          src: url,
```

Comment:
Might warrant a comment here noting that we are passing `url` and not `raw` on purpose and why.